CONTRIBUTING.md

```diff
@@ -0,0 +1,124 @@
+# Open Source by Pieces Contributor Guidelines
+
+We are beyond excited that you want to contribute! We would love to accept your contributions. Pieces Open Source is built for the community and warmly welcomes collaboration. There are many ways in which one could contribute to Pieces Open Source, and every contribution is equally appreciated here. Navigate through the following to understand more about contributing.
+
+## General Contribution Flow
+
+### Contributing Guidelines
+
+Firstly, thank you for considering contributing to Open Source by Pieces. This guide details the general information that one should know before contributing to the repository.
+Please stick as closely as possible to the guidelines. That way, we ensure that you have a smooth experience contributing to this project.
+
+
+### General Rules:
+
+These are, in general, rules that you should be following while contributing to an Open-Source Pieces project:
+
+- Be Nice, Be Respectful (BNBR)
+- Check if the Issue you created already exists or not
+- When creating a new issue, make sure you describe the issue clearly
+- Use tags to mark your issue as “draft” (we will remove the draft tag once we assign the proper issue tags to the created issue)
+- Make proper commit messages and document your PR well
+- Always add comments to your code
+- Follow proper code conventions because writing clean code is important
+- Add @jordan-pieces as a reviewer to be assigned to another maintainer or to be reviewed
+- Issues will be assigned on a "First Come, First Served" basis
+- Once an issue has been assigned, if we don’t receive any reply/response in a week, we will reassign the issue to someone else and remove the assignee
+- Do mention (@jordan-pieces) the project maintainer if your PR isn't reviewed within three days
+
+## Getting Started
+
+### Pre-requisites
+
+1. Ensure you have the latest version of Pieces OS installed on your system:
+   - [Windows](https://docs.pieces.app/installation-getting-started/windows)
+   - [macOS](https://docs.pieces.app/installation-getting-started/macos)
+   - [Linux](https://docs.pieces.app/installation-getting-started/linux)
+
+2. Ensure that you have [NPM](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm) installed.
+
+### Pull Request
+
+**1.** Fork the [repository](https://github.com/pieces-app/example-ts) on GitHub.
```

Comment:
Update this for this repo.

Reply:
Just updated it. Thanks for catching this error.